noxfile.py

```diff
@@ -40,6 +40,26 @@ def docs(session):
     )
 
 
+# Linting with flake8.
+#
+# We ignore the following rules:
+#   E203: whitespace before ‘:’
+#   E266: too many leading ‘#’ for block comment
+#   E501: line too long
+#   I202: Additional newline in a section of imports
+#
+# We also need to specify the rules which are ignored by default:
+# ['E226', 'W504', 'E126', 'E123', 'W503', 'E24', 'E704', 'E121']
+FLAKE8_COMMON_ARGS = [
+    "--show-source",
+    "--builtin=gettext",
+    "--max-complexity=20",
+    "--exclude=.nox,.cache,env,lib,generated_pb2,*_pb2.py,*_pb2_grpc.py",
+    "--ignore=E121,E123,E126,E203,E226,E24,E266,E501,E704,F401,W503,W504,I202",
+    "--max-line-length=88",
+]
```

Comment:
In other repositories, we put this kind of configuration in a `.flake8` file in the root.  E.g., see the [`python-bigtable` version](https://github.com/googleapis/python-bigtable/blob/master/.flake8).